EducationPortal.DAL/Identity/AccountManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using EducationPortal.DAL.Entities;
+using EducationPortal.DAL.FS;
+using EducationPortal.DAL.Infrastructure;
+
+namespace EducationPortal.DAL.Identity
+{
+    public class AccountManager
+    {
+        private IdentityContext db;
```

Comment:
You should use Interface for db (or repository) and inject it

Reply:
Changed to interface